package.json

```diff
@@ -12,7 +12,8 @@
     "test-unit": "sh scripts/tests/testUnit.sh",
     "test": "yarn lint && yarn test-unit && yarn test-integration",
     "tdd:watch": "sh scripts/tests/tdd.sh",
-    "generate-api-schema": "node utils/generateAPISchema.js"
+    "generate-api-schema": "node utils/generateAPISchema.js",
+    "pretty": "prettier --write \"./**/*.{js,jsx,json}\""
```

Comment:
This is already done by `lint-fix`, right?

Reply:
Okay  I did not notice the lint fix  cmd ? I will remove the prettier cmd